style/style.css

```diff
@@ -418,102 +417,39 @@ footer {
   -webkit-transition: 2s;
   -moz-transition: 2s;
   transition: 2s; }
-
-.footer-socials-logos:hover {
-  background-color: #00c4cc;
-  -webkit-transform: rotate(-360deg);
-  -moz-transform: rotate(-360deg);
-  transform: rotate(-360deg); }
+  .footer-socials-logos:hover {
+    background-color: #00c4cc;
+    -webkit-transform: rotate(-360deg);
+    -moz-transform: rotate(-360deg);
+    transform: rotate(-360deg); }
 
 .footer-menu {
   width: 100%;
   list-style-type: none;
   margin: 0px;
   padding: 0px; }
-
-.footer-menu li {
-  display: inline-block;
-  text-align: center; }
-
-.footer-menu a {
-  display: block;
-  text-align: center;
-  padding: 10px 30px;
-  margin: 15px 10px 0px 10px;
-  text-decoration: none;
-  color: #f5f5f5;
-  font-size: 15px;
-  border-radius: 10px;
-  -webkit-transition: background-color 1s;
-  -moz-transition: background-color 1s;
-  transition: background-color 1s; }
-
-.footer-menu a:hover {
-  background-color: #00c4cc; }
+  .footer-menu li {
+    display: inline-block;
+    text-align: center; }
+  .footer-menu a {
+    display: block;
+    text-align: center;
+    padding: 10px 30px;
+    margin: 15px 10px 0px 10px;
+    text-decoration: none;
+    color: #f5f5f5;
+    font-size: .9375rem;
+    border-radius: 10px;
+    -webkit-transition: background-color 1s;
```

Comment:
Zamień ten transition korzystając z istniejącego @mixin'a w pliku variables.scss. Stworzyłem dla takiego transition mixina.

Reply:
@mixin hover-transition(); rozumiem że to jest to ,tylko wydaje mi sie że dotyczy to też hovera a tam akurat go nie ma.

Reply:
odpisałem na Slacku.